src/api/apiMethods.js

```diff
@@ -153,39 +260,62 @@ const getApiMethodsSelector = () =>
         return columnGroup;
       };
 
-      const getColumn = (
-        columnIndex,
-        cellGroupType = CellGroupType.SCROLLABLE
-      ) => {
+      const getColumn = (columnIndex) => {
+        validateColumnIndex(columnIndex);
```

Comment:
Add such validator method in exposed apis to ensure every helper method always receive valid inputs irrespective of arguments passed by client while calling public apis.

Reply:
Nice, I like this.

Let's backport this to v2.0-beta too, with a followup review.